### PR TITLE
Task/sapnamysore/tlt 2645/fix breadcrumb style

### PR DESCRIPTION
**Pull request by sapnamysore**
Removed the custom lti-css/bulkSiteCreator.css that was in place that had partial styles from bootstrap and then some custom ones. Swapped it with standard bootstrap.css in all screens. Moved from styles from bulkSiteCreator.css to app.css to fix some positioning styles.

The screens in Canvas Site Creator are now more consistent with the rest of teh tools in admin console. Any custom styles can be fixed or added as enhancement once @Vittorio2015 and the PO take a look.

The code is deployed on Dev https://canvas.dev.tlt.harvard.edu/accounts/8/external_tools/44

### CODE REVIEW

**Comment by elliottyates** [comment]
Hmm, I would prefer to approach this by standardizing the styles in this app with the ones in the other apps -- e.g. use bootstrap.css in all pages (it's currently not included on these two), and removing the `lti-css/bulkSiteCreator.css` in `base.html`, so there aren't conflicts. It would be very hard otherwise to make everything work (for instance, at the moment there are other subtle differences between the headers in these two pages and the other three in this app -- like the link underlining, explicit background coloring, etc). Switching to a common set of styles and removing the extraneous overrides (much of the `lti-css/bulkSiteCreator.css` is following our previous customized bootstrap stuff, and can be swapped out safely for the regular packaged bootstrap CSS) will mean our headers, datatable formats, etc will be consistent across all apps in this project as well.

While we're at it, in general these CSS/JS blocks can all be cleaned up, it looks like. Each page has implemented its additions to base.html a little differently. I think making them consistent and cutting out the `lti-css/bulkSiteCreator.css` does mean a few things (like padding) will change, but we can fix that up much easier by editing `app.css` at that point and adding useful classes to target things so they don't conflict with shared CSS.

I'm happy to take a first pass at that. If we want to proceed with tweaking these two pages, let's try and get most of the computed CSS properties of the header text to be (a) consistent across all five screens in the app, and (b) consistent with other apps in the project.

**Comment by sapnamysore** [comment]
Sure, good with switching out the custom bootstrap to the standard one. The introduction of the standard bootstrap messed up pretty much everything in the 2 screens..

File: canvas_site_creator/templates/canvas_site_creator/base.html [hunk 1]
```diff
@@ -7,9 +7,16 @@
...
+    <link rel="stylesheet" type="text/css" href="{% static 'bootstrap-3.3.6/dist/css/bootstrap.min.css' %}" media="screen"/>
+
+    {% block css %}
+      {% if debug %}
+        <link rel="stylesheet" type="text/css" href="{% static 'bootstrap-3.3.6/dist/css/bootstrap.css' %}" media="screen"/>
+      {% else %}
+        <link rel="stylesheet" type="text/css" href="{% static 'bootstrap-3.3.6/dist/css/bootstrap.min.css' %}" media="screen"/>
+      {% endif %}
+    {% endblock css %}
 {% if debug %}
```
Comment (elliottyates) [inline canvas_site_creator/templates/canvas_site_creator/base.html]: As per the issue Chrissy reported, if we get the bootstrap JS into this block, and remove the tooltip.js and tooltip.css stuff from the course selection page, that appears to fix the weird modal behavior.

**Comment by elliottyates** [comment]
One breaking change that PO found, and some suggestions. Probably worth another quick functionality test after that -- and maybe just cleaning up the whole CSS/JS on each of the five pages in general (taking out custom JS/CSS and consolidating standard stuff in base.html).

**Comment by elliottyates** [comment]
Looks great! A few suggestions for cleanup but so nice to see this in better shape than it was in icommons_lti_tools.

**Pull request merged**